eventhubs/client/azure-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventDataBatchTest.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.amqp.exception.ErrorCondition;
+import com.azure.core.amqp.exception.ErrorContext;
+import com.azure.messaging.eventhubs.implementation.ErrorContextProvider;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.mockito.Mockito.when;
+
+public class EventDataBatchTest {
+    private static final String PARTITION_KEY = "PartitionIDCopyFromProducerOption";
+
+    @Mock
+    private ErrorContextProvider errorContextProvider;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void nullEventData() {
+        final EventDataBatch batch = new EventDataBatch(1024, PARTITION_KEY, null);
+        batch.tryAdd(null);
+    }
+
+    /**
+     * Verify that if we try to add a payload that is too big for the EventDataBatch, it throws.
```

Comment:
Incomplete comment, This is my mistake. ", it throws" can be removed.

Reply:
It makes sense in this case.